src/test/resources/config/application-artemis.yml

```diff
@@ -3,7 +3,8 @@
 # ===================================================================
 
 artemis:
-    course-archives-path: ./exports/server-integration-test/courses         # a folder in which archived courses and exams are stored.
+    course-archives-path: ./exports/server-integration-test/courses # a folder in which archived courses and exams are stored.
+    legal-path: ./legal/server-integration-test                     # a folder in which legal files (privacy statement, imprint) are stored.
```

Comment:
Should probably be added to the gitignore, as it is marked as changed otherwise after running tests locally?